src/Microsoft.Health.Fhir.Stu3.Api/Features/Operations/OperationsCapabilityProvider.cs

```diff
@@ -0,0 +1,67 @@
+// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using Microsoft.Health.Extensions.DependencyInjection;
+using Microsoft.Health.Fhir.Core.Features.Conformance;
+using Microsoft.Health.Fhir.Core.Features.Conformance.Models;
+using Microsoft.Health.Fhir.Core.Features.Operations;
+using Microsoft.Health.Fhir.Core.Features.Search;
+using Microsoft.Health.Fhir.Core.Models;
+
+namespace Microsoft.Health.Fhir.Api.Features.Operations
+{
+    /// <summary>
+    /// Class that handles FHIR STU3 specific behavior for adding export operation
+    /// details to the capability statement.
+    /// </summary>
+    public partial class OperationsCapabilityProvider
+    {
+        private void AddExportDetailsHelper(ICapabilityStatementBuilder builder)
+        {
+            builder.Apply(AddExportDetails);
+        }
+
+        public void AddExportDetails(ListedCapabilityStatement capabilityStatement)
+        {
+            GetAndAddOperationDefinitionUriToCapabilityStatement(capabilityStatement, OperationsConstants.Export);
+            GetAndAddOperationDefinitionUriToCapabilityStatement(capabilityStatement, OperationsConstants.PatientExport);
+            GetAndAddOperationDefinitionUriToCapabilityStatement(capabilityStatement, OperationsConstants.GroupExport);
+        }
+
+        public void AddPatientEverythingDetails(ListedCapabilityStatement capabilityStatement)
```

Comment:
>AddPatientEverythingDetails [](http://example.com/codeflow?start=20&length=27)

this is different in main, can you remove this file?

Reply:
This file is not part of the GraphQL work, while doing a merge with the main branch, github took the new files from others as my changes.